omod/src/main/resources/config.xml

```diff
@@ -138,6 +138,13 @@
 	</messages>
 	<!-- /Internationalization -->
 
+	<globalProperty>
+		<property>registrationapp.paperRecordIdentifierDefinition</property>
+		<description>
```

Comment:
for backwards compatibility, the default value here should be the one that is currently used ""paperrecord.patientDataDefinition.paperRecordIdentifier"